beacon_node/beacon_chain/tests/store_tests.rs

```diff
@@ -743,7 +743,7 @@ fn prunes_abandoned_fork_between_two_finalized_checkpoints() {
     let (stray_blocks, stray_states, _, stray_head, _) = harness.add_stray_blocks(
         harness.get_head_state(),
         slot,
-        slots_per_epoch - 1,
+        slots_per_epoch - 3,
```

Comment:
I can't quite work out why this needs to be `- 3`, can you remind me?

Reply:
It's in the commit messages.